tests/test_sklearn_gbdt_converter.py

```diff
@@ -169,6 +169,28 @@ def test_float64_GB_trees_regressor_converter(self):
             self.assertIsNotNone(torch_model)
             np.testing.assert_allclose(model.predict(X), torch_model.predict(X), rtol=1e-06, atol=1e-06)
 
+    def test_varying_batch_sizes(self):
```

Comment:
Should we get a similar test for `onnx`? This already worked before because torch uses dynamic dimensions by default

Reply:
nice catch, added logic checks in a few more places